tests/Http/Jobs/ReadRestControllerTest.php

```diff
@@ -44,7 +44,7 @@ public function it_returns_a_json_response_for_existing_job()
 
         $expectedResponse = new JsonResponse($jobStatus->toNative());
 
-        $this->assertEquals($expectedResponse, $response);
+        $this->assertEquals($expectedResponse->getContent(), $response->getContent());
```

Comment:
Yep this used to happen a lot in the past. We always fixed it like this but I guess some hadn't been changed yet.